addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiClientThingHandler.java

```diff
@@ -111,15 +114,34 @@ public void initialize() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // mgb: only handle the command if we're ONLINE
-        if (getThing().getStatus() == ONLINE) {
-            logger.debug("Handling command = {} for channel = {}", command, channelUID);
-            if (command == REFRESH) {
+        logger.debug("Handling command = {} for channel = {}", command, channelUID);
+
+        if (command instanceof OnOffType) {
```

Comment:
Put this block in a separate method, just like is done with the `refreshChannel` call.

Reply:
Done.